guideline.md

```diff
@@ -0,0 +1,126 @@
+
+The Approval Group has the task not only of approving or denying requests but also of keeping track of the balance of the fund and encouraging its full use each year. We know that some chapters are more active or are simply located in larger cities and have more members, so they may need to apply for a larger portion of the fund. However, it is important to keep in mind that this funding can be the incentive needed to support smaller groups to be more active or to provide the tools and means needed for their members to be more active. Similarly, this concept should prevail when it comes to using funds to encourage support for marginalized groups within PyLadies, so that it reaches those that need more support or structure.
+
+## PyLadies Germany Spokesperson
+
+PyLadies Germany will have a spokesperson at the PySV to mediate between the two groups and inform other chapters (existing and new) about the process. This person is responsible for supporting chapters to follow these guidelines and tracked expenses are reported. They are not responsible to singularly deliver these results as this is a shared responsibility of all chapters. If they feel that the process is being misused or have any other concerns, they should raise it with the rest of the approval group for discussion and the PySV.
+
+The role should rotate and is open to any PyLady in Germany. A mechanism for this rotation and the period of tenure is not yet defined. 
+
+As this person will be from one of the chapters represented in the approval group, they should refrain from being the second approver if the other representative of their chapter has already approved.
+PyLadies Germany Spokesperson attendance of PySV meetings:
+
+The spokesperson's involvement will also include attending PySV meetings. The frequency and schedule of these meetings will be determined jointly by PyLadies and PySV.
+
+## Reporting:
+
+The platform for approving and tracking requests should be as publicly open as possible so that any PyLady in Germany has access. Therefore no individual's personal information should be shared unless it is relevant (individuals can be referred to by their role i.e. event speaker). Invoices should not be posted there.
+
+Tickets will be labeled by chapter and year and the platform should allow PyLadies to see at a glance what has been requested, approved, and reimbursed. Anyone should be able to make a request and there should be guidelines on how to do so.
+
+A general update on notable events and activities should be sent to PySV every six months. This helps PySV understand the needs and accomplishments of PyLadies. The report should be sent by the spokesperson with the support of the other approval group members.
+
+Details of all expenditures must be reported so that accounts can be properly reconciled. These are sent directly to the PySV with the germany@pyladies.com email in CC.
```

Comment:
```suggestion
Details of all expenditures must be reported so that accounts can be properly reconciled. These are sent directly to the PySV with the [pyladies@python-verband.org](mailto:pyladies@python-verband.org) email in CC.
```